tests/data_context/cloud_data_context/test_datasource_crud.py

```diff
@@ -12,7 +12,7 @@
 
 
 @pytest.mark.cloud
-@pytest.mark.integration
+@pytest.mark.e2e
```

Comment:
@cdkini I think these tests should still be marked `cloud` and `integration` since they don't actually hit the cloud service and merely mock the calls that are expected.